### DISCUSSION TITLE
0.5.9

### DISCUSSION THREAD

**Discussion by z-huang** [z-huang/music#1478] [Announcements]
- **Revamped home screen**
- Multi-select in queue and playlists
- Add queue item menu
- Search in playlists
- Option to change grid cell size
- Option to stop music when the app is killed
- Option to auto skip to next song on error
- Click on bottom navigation item again to scroll to top
- Refetch song button
- Fix repetitions in history
- Other fixes and enhancements

<hr /><em>This discussion was created from the release <a href='https://github.com/z-huang/InnerTune/releases/tag/v0.5.9'>0.5.9</a>.</em>

**Comment by Roxo7777** [comment]
Thank 🫶🏻

**Reply by JoelJaks** [reply]
Just downloaded this app, I have been looking for an alternative to ymusic which will keep the memory of last paused at such that I don't have to search every time I come back to continue, but just press play. Am yet to see if this will be the solution but will come back with a comment after trying it out. Thanks!!

**Comment by FL1kkeR** [comment]
Thank you for adding the search in playlist function, however, it doesn't work quite well, whenever you search and play the song, the app will run the first song in the playlist and not the one you searched for, it happens 100% of the time.

**Reply by z-huang** [reply]
Sorry for the mistake. I made a commit to fix it.

**Reply by FL1kkeR** [reply]
Best app I've ever used, been using Spotify for the last 5 years, yesterday I stumbled upon this app, you have created the perfect music streaming app, will support you as much as I can, thank you for updating the app regularly and fixing the bugs!

**Comment by Roxo7777** [comment]
Would it be possible in the next version to add the multiple selection of songs in the from downloaded tab? So I can put them in a playlist

**Reply by z-huang** [reply]
Okay.

**Reply by Anonuser-code** [reply]
+1

**Comment by kaushiknath44** [comment]
Please try to add loop option between 2 points in music feature in the next update and there is a majo big in animation in the home tab ; whenever I go from any tab like songs playlist etc to home tab it has a major frame drop issue

**Reply by z-huang** [reply]
> Please try to add loop option between 2 points in music feature

Do you mean AB loop? I won't personally work on that because it's not needed for a normal music listener.

**Reply by kaushiknath44** [reply]
Yeah ab loop 
Please try to add if possible becoz I m completely dependent on this app

**Comment by ruskcoder** [comment]
How can I play a playlist and having it "radio" after the playlist is completed? So when the playlist finishes, it finds song recommendations and the queue doesn't end. 
Also, can we have a 5 second move forward animation when the thumbnail is double pressed? (Like the YouTube Videos app, double press shows a little animation)

**Comment by unseenlarks** [comment]
thank you.

**Comment by Melo-5** [comment]
Thanks for the update Z!

Is it possible to add the option (play next) in the queue menu?

**Reply by z-huang** [reply]
Done.

**Reply by Melo-5** [reply]
You're the best! Keep it up!

**Comment by chavishcb** [comment]
Thank you. Can we have the album art on songs, saved playlists which are downloaded, while we are offline too? It would be easier for navigation.

**Comment by aaru111** [comment]
@z-huang so when I play a song and close the app like in the photo
![Screenshot_20240829-233728](https://github.com/user-attachments/assets/500d47ed-ef4c-4bbb-ad6d-1cb754832a8f)

the app doesn't show the last song that was playing it shows the previous song before the last song if you get what I mean

please look into it

**Comment by thornySoap** [comment]
@z-huang Hello, posting here because creating a pull request or issue for that seamed a bit overkill. I noticed when creating a pull request at Malopieds' fork that the sleep timer icon wasn't aligned to the middle which was more noticable there because the button is tinted. But looking at the ripple effect in the original app it doesn't look aligned, too. If that's not intended redownloading the vector asset should do it.

**Reply by z-huang** [reply]
Fixed. You have a keen eye 😄

**Comment by nialthony** [comment]
Keep your great work, I love it so much 🫶🏻

**Comment by AminJnr** [comment]
@z-huang Can you add the option to play the video of the song like simpmusic if possible?🙏

**Reply by z-huang** [reply]
InnerTune is a music player. I don't think it necessary to play videos. Also, it's much more complicated to add video support.  So, I won't work on it, but contribution is welcome.

**Comment by Snapxxx12** [comment]
Is there a desire to add audio fade-in and fade-out?

**Comment by Medoppss** [comment]
![Screenshot_20250502_015138](https://github.com/user-attachments/assets/8dde10c3-4f5e-4bad-8686-2daa8c44832f)
Why it doesn't work with me 😞😭
